docs/install/install.md

```diff
@@ -21,21 +21,23 @@ limitations under the License.
 
 
 
-## From binary package
+## Zeppelin Installation
+Welcome to your first trial to explore Zeppelin ! 
 
-   Download latest binary package from [Download](../download.html).
+In this documentation, we will guide you how you can install Zeppelin from **Binary Package** or build from **Source** by yourself. Plus, you can get a specific infomation about Zeppelin configurations at the below **Zeppelin Configuration** section.
```

Comment:
Thank you for the review @corneadoug : ) 
I applied your comments and pushed again.